features/admin.sms-providers.v1/pages/twilio-sms-provider.tsx

```diff
@@ -42,6 +43,22 @@ const TwilioSMSProvider: FunctionComponent<TwilioSMSProviderPageInterface> = (
         isReadOnly
     } = props;
     const { t } = useTranslation();
+    const [ isShow, setIsShow ] = useState(false);
+
+    const renderInputAdornment = (): ReactElement => (
+        <InputAdornment position="end">
+            <Icon
```

Comment:
Shall we use eyeIcon and eyeSlashIcon from oxygen-ui/react-icons for consistency :)?

https://wso2.github.io/oxygen-ui/react/?path=/docs/icons-icons-%F0%9F%9A%A7--page

Reply:
This was checked before https://github.com/wso2/identity-apps/pull/5935#discussion_r1565166693